magento2-varnish/4.0/usr/local/share/env/30-framework

```diff
@@ -2,3 +2,16 @@
 
 export MAGENTO_MODE=${MAGENTO_MODE:-production}
 export PURGE_IPS=${PURGE_IPS:-\"172.17.0.0\"/16,\"172.20.0.0\"/16,\"10.0.0.0\"/8}
+
+MAGENTO_USE_SEPARATE_ADMIN_CONTAINER=${MAGENTO_USE_SEPARATE_ADMIN_CONTAINER:-false}
+MAGENTO_USE_SEPARATE_ADMIN_CONTAINER="$(convert_to_boolean_string "$MAGENTO_USE_SEPARATE_ADMIN_CONTAINER")"
+export MAGENTO_USE_SEPARATE_ADMIN_CONTAINER
+
+export MAGENTO_ADMIN_FRONTNAME_REGEX_ESCAPED=${MAGENTO_ADMIN_FRONTNAME_REGEX_ESCAPED:-admin}
+
+export MAGENTO_ADMIN_BACKEND_HOST=${MAGENTO_ADMIN_BACKEND_HOST:-admin}
+export MAGENTO_ADMIN_BACKEND_PORT=${MAGENTO_ADMIN_BACKEND_PORT:-80}
```

Comment:
we default to https-only in the base images, do the magento2 images turn that off?